google/cloud/storage/examples/storage_quickstart.cc

```diff
@@ -33,20 +33,21 @@ int main(int argc, char* argv[]) try {
   // uses the default configuration for authentication and project id.
   gcs::Client client;
 
-  gcs::BucketMetadata metadata =
-      client
-          .CreateBucketForProject(
-              bucket_name, project_id,
-              gcs::BucketMetadata()
-                  .set_location("us-east1")
-                  .set_storage_class(gcs::storage_class::Regional()))
-          .value();
+  google::cloud::StatusOr<gcs::BucketMetadata> bucket_metadata =
+      client.CreateBucketForProject(
+          bucket_name, project_id,
+          gcs::BucketMetadata()
+              .set_location("us-east1")
+              .set_storage_class(gcs::storage_class::Regional()));
```

Comment:
Add a new line here.

Reply:
You want a second one?  There is a blank line before the `if (!bucket_metadata)` statement.

Reply:
Acking, I missed that. Please ignore this comment.